manifests/f/FreakinSoftMania/WinUEFI/v1.9.2/FreakinSoftMania.WinUEFI.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FreakinSoftMania.WinUEFI
+PackageVersion: v1.9.2
+PackageLocale: en-US
+Publisher: FreakinSoftMania

```

```suggestion
Publisher: Minionguyjpro
```
This must match Control Panel.

---

What do you mean, control panel? I changed this for a reason. I rebranded it from myself to that GutHub organization.

---

What should I do? Make a new release first with the updated publisher? I changed it in the setup script (ISS). I know this is also responsible for making the control panel entry at programs. I woll do that and change it a little again after that.